### PR TITLE
Added card "Exuberant Fuseling" and its abilities and effects

### PR DESCRIPTION
**Pull request by Merlingilb**

### CODE REVIEW

File: Mage.Sets/src/mage/cards/e/ExuberantFuseling.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        this.addSubType(SubType.GOBLIN);
+        this.addSubType(SubType.WARRIOR);
+        this.power = new MageInt(0);
+        this.toughness = new MageInt(1);
+
+        //Trample
+        this.addAbility(TrampleAbility.getInstance());
+
+        //Exuberant Fuseling gets +1/+0 for each oil counter on it.
+        this.addAbility(new SimpleStaticAbility(new BoostSourceEffect(oilCounters, oilCounters, Duration.WhileOnBattlefield)));
```
Comment (PurpleCrowbar) [inline Mage.Sets/src/mage/cards/e/ExuberantFuseling.java:40]: This boost effect should only increase power, not toughness, currently incorrectly grants +1/+1 for each oil counter on the permanent

Comment (PurpleCrowbar) [inline Mage.Sets/src/mage/cards/e/ExuberantFuseling.java:40]: Please apply `.setText("{this} gets +1/+0 for each oil counter on it")` to `BoostSourceEffect` for desired rule text

File: Mage.Sets/src/mage/cards/e/ExuberantFuseling.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+        //When Exuberant Fuseling enters the battlefield and whenever another creature or artifact you control is put
+        //into a graveyard from the battlefield, put an oil counter on Exuberant Fuseling.
+        this.addAbility(new OrTriggeredAbility(Zone.BATTLEFIELD, new AddCountersSourceEffect(CounterType.OIL.createInstance()),
```
Comment (PurpleCrowbar) [inline Mage.Sets/src/mage/cards/e/ExuberantFuseling.java:44]: Please apply `.setTriggerPhrase("When {this} enters the battlefield and whenever another creature or artifact you control is put into a graveyard from the battlefield, ")` to `OrTriggeredAbility` for desired rule text

**Review by PurpleCrowbar — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/e/ExuberantFuseling.java (hunk 2 above)
Comment (Merlingilb) [reply Mage.Sets/src/mage/cards/e/ExuberantFuseling.java:46]: done

File: Mage.Sets/src/mage/cards/e/ExuberantFuseling.java (hunk 1 above)
Comment (Merlingilb) [reply Mage.Sets/src/mage/cards/e/ExuberantFuseling.java]: done

Comment (Merlingilb) [reply Mage.Sets/src/mage/cards/e/ExuberantFuseling.java]: done (bad copy and paste ;)

**Pull request merged**